OloEngine/src/OloEngine/Renderer/Renderer3D.cpp

```
@@ -37,12 +39,81 @@ namespace OloEngine
+    }
+
+    // Helper to create default POD render state
+    static PODRenderState CreateDefaultPODRenderState()
+    {
+        PODRenderState state{};
+        // All fields are initialized to sensible defaults by the struct itself
+        return state;
+    }
+
+    // Helper to populate POD render state from material and optional overrides
+    static PODRenderState CreatePODRenderStateForMaterial(const Material& /*material*/)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)